tests/primeqa/text_classification/processors/preprocessors/test_textclassifier_preprocessor.py

```diff
@@ -0,0 +1,41 @@
+from transformers import AutoTokenizer
+from primeqa.mrc.data_models.target_type import TargetType
+from primeqa.text_classification.processors.preprocessors.text_classifier import TextClassifierPreProcessor
+#from primeqa.boolqa.processors.preprocessors.boolqa_classifier import BoolQAClassifierPreProcessor
+from primeqa.boolqa.processors.dataset.mrc2dataset  import create_dataset_from_run_mrc_output
+from tests.primeqa.mrc.common.base import UnitTest
+
+
+
+class TestNWayPreProcessor(UnitTest):
+    _eval_pred_in_file = 'tests/resources/text_classification/processors/preprocessors/eval_predictions.json'
+    _expected_eval_dataset0 = {'example_id': '9d854612-d484-4b98-b208-4983baeeab67',
+                              'question': 'Do zebra finches have stripes?',
+                              'label': 0,
+                              'span_answer_text': 'There are two subspecies of the zebra finch:',
```

Comment:
It probably doesn't matter, but this answer is not correct?

Reply:
copied from the corresponding boolean - its just a regression test to catch if the preprocessor output changes for the same input